src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/DraftApprenticeshipController.cs

```diff
@@ -62,6 +68,11 @@ public async Task<IActionResult> Get(long cohortId, long apprenticeshipId)
 
             var response = await _mediator.Send(command);
 
+            if (response == null)
+            {
+                return NotFound();
+            }
+
```

Comment:
This seems sensible, just curious what happens in the client. I assume it converts it to a null